pipelines/pom.xml

```diff
@@ -375,6 +375,10 @@
           <compilerArgs>
             <!-- Enable access to the packages at compile time -->
             <arg>--add-exports=java.base/jdk.internal.misc=ALL-UNNAMED</arg>
+            <!-- To retain parameter names for compiled classes, this is needed for latest spring.
```

Comment:
nit: please mention the version of Spring after which it is needed (instead of "latest").

Reply:
Done